CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
 - Commands - Remove command run and install.
 - Command Generate - Install Forest Admin middleware.
 
+### Fixed
+- Command Login - Fix login only enable connection (And not account creation).
+- Command Login - Verify the validity of the token passed in paramaters.
```

Comment:
If you have better ideas on the changelog, feel free

Reply:
```suggestion
- Command Login - Check token validity before login.
```